src/components/Inputs/Schemas.js

```diff
@@ -0,0 +1,135 @@
+import { Button } from '@dhis2/ui-core'
```

Comment:
This file contains a lot of components, would be more understandable if it were broken up, say along the lines of the SelectAll, SelectNone controls and the components display the actual schemas for example.

Reply:
I've split it into 1. Schema, 2. Schema/Controls and 3. Schema/SchemaGroup